cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -241,12 +306,13 @@ void ImageFlip::imageCallback(const sensor_msgs::ImageConstPtr& color_image_msg)
 		for (int v = 0; v < color_image_turned.rows; v++)
 			for (int u = 0; u < color_image_turned.cols; u++)
 				color_image_turned.at<cv::Vec3b>(v,u) = color_image.at<cv::Vec3b>(u,color_image.cols-1-v);
-		rot_mat.at<double>(0,1) = 1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.